pages/index.js

```diff
@@ -1,9 +1,112 @@
-import EmotionList from "@/components/EmotionList";
+import styled from "styled-components";
+import useLocalStorageState from "use-local-storage-state";
+import { uid } from "uid";
+import Link from "next/link";
+import { date } from "@/utils/date-constructor";
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  width: 80vw;
+  margin: 2rem auto;
+  justify-content: center;
+  align-items: center;
+`;
+
+const StyledLabel = styled.label`
+  padding: 2rem;
+  text-align: center;
+`;
+
+const StyledInput = styled.input`
+  width: inherit;
+`;
+
+const StyledWrapper = styled.div`
+  display: flex;
+  width: inherit;
+  justify-content: space-between;
+`;
+
+const StyledSpan = styled.span`
+  font-size: 0.8rem;
+`;
+
+const StyledButton = styled.button`
+  width: 3rem;
+`;
+
+const StyledList = styled.ul`
+  list-style: none;
+  padding: 0;
+  margin: 2rem auto;
+  text-align: left;
+`;
+
+const StyledListItem = styled.li`
+  margin: 1.2rem auto;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  padding: 1rem;
+  background-color: lightgray;
+  border: 1px solid black;
+  border-radius: 5px;
+  color: black;
+`;
 
 export default function HomePage() {
+  const [tensionEntry, setTensionEntry] = useLocalStorageState("tensionEntry", {
+    defaultValue: [],
+  });
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    // variable date imported from util/date-constructor.js
+    setTensionEntry(
+      tensionEntry.length === 0
+        ? [{ ...data, id: uid(), date: date }]
+        : [...tensionEntry, { ...data, id: uid(), date: date }]
+    );
+  }
```

Comment:
- Please note my comment regarding the timestamp in the file [utils/date-constructor.js](https://github.com/janaRicarda/emotion-tracker/pull/13/files#diff-e02ce9ec39dfdda425aa10006a2364724a9e7bea55dad586e941743cdcd87af3)
- It's generally a good idea to separate concerns by handling the state update logic in a separate function from your event handler.

Example:
```js
function handleAddTensionEntry (data) {
   // function to use the current time and date ⬇️ 
    const entryDate = getCurrentFormattedDate(); 
    const newEntry = { ...data, id: uid(), date: entryDate };

    setTensionEntry(prevEntries => [
      ...prevEntries,
      newEntry
    ]);
  };

function handleSubmit(event) {
    event.preventDefault();
    const formData = new FormData(event.target);
    const data = Object.fromEntries(formData);

    addTensionEntry(data);
  }
```